gkeonprem/gkeonprem_baremetal_cluster_metallb/main.tf

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#[START gkeonprem_bare_metal_cluster_metallb]
+
+# GCP project information is fetched from the 'GOOGLE_CLOUD_PROJECT' environment
+# variable if it is not explicitely set in this data block.
+data "google_project" "project" {
+  project_id = "YOUR_PROJECT_ID"
+}
+
+# Fill in the local variables according to your baremetal environment; you must
+# already have an admin cluster created according to the prerequisites outlined
+# in https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/installing/creating-clusters/create-user-cluster-api#before_you_begin
+locals {
+  admin_cluster     = "YOUR_ADMIN_CLUSTER_NAME"
```

Comment:
Can you reverse this where you put the values in the quotes and the meaning of the value in the comments?

My hope is that by doing this the sample will run as-is and customers can replace the hard-coded values with their own values.